docs/devguide/repl.rst

```diff
@@ -7,16 +7,25 @@ Accessing the REPL
 REPL (Read-Evaluate-Print-Loop) allows the micro:bit to read and evaluate code 
 in real-time as you write it.
 
-Accessing the REPL on the micro:bit will require you to:
+.. note::
 
-* Determine the communication port identifier for the micro:bit
-* Use a program to establish communication with the device
+    For Windows 7 you might need to install the 
+    `MBED serial driver <https://os.mbed.com/docs/mbed-os/v6.8/program-setup/windows-serial-driver.html>`_.
 
-For versions of Windows before 10 you might need to install the Mbed serial 
-driver, the instructions for which are found here:
+Using the micro:bit Python Editor
+---------------------------------
+The browser-based Python editor has buil-in REPL support, that can be accessed
+using `WebUSB <https://developers.google.com/web/updates/2016/03/access-usb-devices-on-the-web>`_.
```

Comment:
Do we have any support articles that explains this in the context of the online editors?